lab5.cpp

```diff
@@ -0,0 +1,204 @@
+﻿#include <iostream>
+#include <string>
+using namespace std;
+class Candidate {
+
+private:
+	string name;
+	int count;
+	float percent;
+
+
+
+
+
+
+public:
+	int sum;
+
+
+	string GetName() {
```

Comment:
getName or get_name - only this two version of method are correct

Reply:
it`s getter, not method